### PR TITLE
Renames use-a-faucet. Adds devnet steps.

### PR DESCRIPTION
**Pull request by johnnymatthews**
This PR:

- Renames _Use a faucet_ to _Get test funds_. This is to increase the versatility of this page, essentially telling users how to get funds from a variety of networks.
- Adds steps for how to get FIL on a local testnet.

Closes https://github.com/filecoin-project/filecoin-docs/issues/1664

### CODE REVIEW

File: content/en/developers/smart-contracts/how-tos/deploy-a-contract/index.md [hunk 1]
```diff
@@ -63,7 +63,7 @@ You must have the following installed:
 - [Git](https://git-scm.com/)
 - [Yarn](https://yarnpkg.com/)
 
-You should also have an address on the Filecoin Hyperspace testnet. See the [Add to MetaMask page]({{< relref "add-to-metamask" >}}) for information on how to get an address. You also need test-FIL `tFIL` in your wallet. See the [Use a Faucet]({{< relref "use-a-faucet" >}}) page for information on how to get test funds.
+You should also have an address on the Filecoin Hyperspace testnet. See the [Add to MetaMask page]({{< relref "add-to-metamask" >}}) for information on how to get an address. You also need test-FIL `tFIL` in your wallet. See the [Use a Faucet]({{< relref "get-test-funds" >}}) page for information on how to get test funds.
```
Comment (longfeiWan9) [inline content/en/developers/smart-contracts/how-tos/deploy-a-contract/index.md:66]: Can we use `get-test-tokens`? 
Tokens make more sense than funds for blockchain projects.

File: content/en/developers/smart-contracts/how-tos/get-test-funds/index.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+draft: false
+images: []
+aliases:
+    - "/developers/smart-contracts/how-tos/use-a-faucet/"
+    - "/fvm/how-tos/use-a-faucet/"
+---
+
+## Hyperspace testnet
+
+Before we begin, you must have an address on the Hyperspace testnet. Follow the [Add to MetaMask guide]({{< relref "add-to-metamask" >}}) if you don't have one yet.
```
Comment (longfeiWan9) [inline content/en/developers/smart-contracts/how-tos/get-test-funds/index.md:20]: Devs can have t4 addresses/ETH addresses, and they can have t1 & t3 addresses as well. Creating a wallet in metamask is one option. Should we also mention that devs can also request testnet tokens using t1&t3 addresses?

File: content/en/developers/smart-contracts/how-tos/deploy-a-contract/index.md (hunk 1 above)
Comment (johnnymatthews) [reply content/en/developers/smart-contracts/how-tos/deploy-a-contract/index.md:66]: Good point. I'll change that.

File: content/en/developers/smart-contracts/how-tos/get-test-funds/index.md (hunk 2 above)
Comment (johnnymatthews) [reply content/en/developers/smart-contracts/how-tos/get-test-funds/index.md:20]: Yeah that's probably good to call out. We don't have to go into a massive amount of detail here, but we can mention that alternative account types, and link to the accounts page.

File: content/en/developers/smart-contracts/how-tos/deploy-a-contract/index.md (hunk 1 above)
Comment (johnnymatthews) [reply content/en/developers/smart-contracts/how-tos/deploy-a-contract/index.md]: Resolved in  38191d95

File: content/en/developers/smart-contracts/how-tos/get-test-funds/index.md (hunk 2 above)
Comment (johnnymatthews) [reply content/en/developers/smart-contracts/how-tos/get-test-funds/index.md]: Resolved in a4988093.

**Pull request merged**